### PR TITLE
fix: TimeSerieType mapping from request

### PR DESCRIPTION
**Pull request by MadsDue**
## Description

Enure we map time serie type correctly

## References

https://app.zenhub.com/workspaces/mosaic-60a6105157304f00119be86e/issues/gh/energinet-datahub/team-phoenix/60
* #0000

### CODE REVIEW

File: source/dotnet/wholesale-api/Events/Events.Application/UseCases/AggregatedTimeSeriesRequestHandler.cs [hunk 1]
```diff
@@ -72,6 +72,15 @@ public async Task ProcessAsync(ServiceBusReceivedMessage receivedMessage, string
             .ToListAsync(cancellationToken: cancellationToken).ConfigureAwait(false);
     }
 
+    private CalculationTimeSeriesType MapTimeSerieType(TimeSeriesType timeSeriesType)
```
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/Events/Events.Application/UseCases/AggregatedTimeSeriesRequestHandler.cs:75]: How do I know whether this implementation is allright? It throws for a lot of known time series types.

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs [hunk 2]
```diff
@@ -113,6 +113,16 @@ private string CreateBatchResultsSql(Guid batchId)
 ";
     }
 
+    private string MapTimeSerieType(TimeSeriesType queryTimeSeriesType)
```
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs:116]: This method throws for a lot of known time series types. I think the method should handle mapping of all types - and tests should ensure this. Also catching future changes to the time series type.

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs [hunk 3]
```diff
@@ -113,6 +113,16 @@ private string CreateBatchResultsSql(Guid batchId)
 ";
     }
```
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs:115]: Looks like there is a candidate extraction regarding mapping or factory...?

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs (hunk 2 above)
Comment (MadsDue) [reply source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs:116]: In order to keep our PR and work small, we have only impl the happy case described on our story. This is why we haven't added the remain yet.

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs (hunk 3 above)
Comment (MadsDue) [reply source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs:115]: Arh I can see that you already has this :)

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs (hunk 2 above)
Comment (MadsDue) [reply source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs:116]: I will use the existings mapper instead which account for them all :)

File: source/dotnet/wholesale-api/Events/Events.Application/UseCases/AggregatedTimeSeriesRequestHandler.cs (hunk 1 above)
Comment (MadsDue) [reply source/dotnet/wholesale-api/Events/Events.Application/UseCases/AggregatedTimeSeriesRequestHandler.cs:74]: In order to keep our PR and work small, we have only impl the happy case described on our story. This is why we haven't added the remain yet.

File: source/dotnet/wholesale-api/Events/Events.UnitTests/Application/UseCases/ProcessAggregatedTimeSeriesRequestHandlerTests.cs
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/Events/Events.UnitTests/Application/UseCases/ProcessAggregatedTimeSeriesRequestHandlerTests.cs:1]: The files seems to be missing a rename

File: source/dotnet/wholesale-api/Events/Events.Application/UseCases/AggregatedTimeSeriesRequestHandler.cs (hunk 1 above)
Comment (BjarkeMeier) [reply source/dotnet/wholesale-api/Events/Events.Application/UseCases/AggregatedTimeSeriesRequestHandler.cs:74]: Ok, accepted 😊
But I anticipate it still makes sense to test at least these two mappings then?

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs (hunk 2 above)
Comment (BjarkeMeier) [reply source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/CalculationResultQueries.cs]: Great idea 💪

File: source/dotnet/wholesale-api/Events/Events.Application/UseCases/AggregatedTimeSeriesRequestHandler.cs (hunk 1 above)
Comment (MadsDue) [reply source/dotnet/wholesale-api/Events/Events.Application/UseCases/AggregatedTimeSeriesRequestHandler.cs]: There u go :)

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**